### DISCUSSION TITLE
Data

### DISCUSSION THREAD

**Discussion by DialFrost** [CliMA/Oceananigans.jl#2228] [General]
Hi, i have project for climate change programming, and i was wondering where the data for the fluid dynamics comes from, as my school states that the data has to come from a scientific instiution. thx!

**Comment by glwagner** [comment]
Hi @DialFrost ! Can you clarify your question? This is a repository for ocean modeling _software_ (not a repository for hosting data). "Data" can mean many things: observational data (derived from observations of the ocean or Earth system), or numerical data (derived from the output of numerical simulations).

**Reply by DialFrost** [reply]
sry i mean where *does* the data come from for the software to create all this graphs

**Reply by navidcy** [reply]
hi @DialFrost, the software generates the data.

**Reply by glwagner** [reply]
To elaborate on @navidcy, if by

> all this graphs

you mean the movies and plots in the [documentation](https://clima.github.io/OceananigansDocumentation/stable/), then the data comes from a numerical simulation that uses the software in this repository. That is, the data represents the numerical solution of partial differential equations that approximately describe the motion of fluids.

What kind of data do you need for your project, apart from that it is vetted by some scientific institution?
